From 38b715192c7be546d0ee0077bc55278676ff8f7b Mon Sep 17 00:00:00 2001
From: Jedediah Smith <jedediah@silencegreys.com>
Date: Fri, 20 Feb 2015 15:23:57 -0500
Subject: [PATCH] Fix playEffect params


diff --git a/src/main/java/org/bukkit/craftbukkit/CraftWorld.java b/src/main/java/org/bukkit/craftbukkit/CraftWorld.java
index 0e9aa7b2..b06d67ec 100644
--- a/src/main/java/org/bukkit/craftbukkit/CraftWorld.java
+++ b/src/main/java/org/bukkit/craftbukkit/CraftWorld.java
@@ -869,7 +869,7 @@ public class CraftWorld implements World {
                 particleFullName.append('_').append(data);
             }
 
-            packet = new PacketPlayOutWorldParticles(CraftEffect.getNMSParticle(effect), false, (float)location.getX(), (float)location.getY(), (float)location.getZ(), offsetX, offsetY, offsetZ, particleCount, radius);
+            packet = new PacketPlayOutWorldParticles(CraftEffect.getNMSParticle(effect), false, (float)location.getX(), (float)location.getY(), (float)location.getZ(), offsetX, offsetY, offsetZ, speed, particleCount);
         }
 
         int distance;
